Framework/PythonInterface/mantid/py3compat/mock.py

```diff
@@ -0,0 +1,15 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+from __future__ import absolute_import
+
+try:
```

Comment:
@martyngigg Do you think this would work? This makes it possible to very easily refactor all existing `from mock import ...` to `from mantid.py3compat.mock import ...`, and PyCharm is still able to auto suggest imports

Reply:
@DTasev This looks good. It mirrors what `qtpy` does as a shim for PyQt :+1: